Tests/Testably.Abstractions.Testing.Tests/FileSystem/FileSystemWatcherMockTests.cs

```diff
@@ -299,7 +299,7 @@ public async Task RenamedEventArgs_ShouldUseDirectorySeparatorFromSimulatedFileS
 			fileSystem.File.WriteAllText(expectedOldFullPath, "foo");
 
 			using IFileSystemWatcher fileSystemWatcher =
-				fileSystem.FileSystemWatcher.New(fileSystem.Path.GetFullPath(parentDirectory));
+				fileSystem.FileSystemWatcher.New(parentDirectory);
```

Comment:
This was added in #866 but I didn't find it necessary. Maybe I'm missing something?

Reply:
This was added on purpose to make the tests succeed after the following observation:
>During testing of #866, I stumbled upon the case that you initialize the file system watcher with a relative directory (e.g. "watched") instead of one with a full path and the returned paths were also relative. Currently the tests fail on the real file system:
> <img width="1478" height="287" alt="Image" src="https://github.com/user-attachments/assets/46336279-b528-463f-9ba0-9702020393b5" />